portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,18 +20,51 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.gson.Gson;
-import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
 
-/** Servlet that returns list of greeting messages. TODO: modify this file to handle comments data */
+/** Servlet that handles comment data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final ImmutableList<String> HELLO_MESSAGES = ImmutableList.of("Hello Megan!", "Hello there!", "Hello!");
+  private ArrayList<Comment> comments = new ArrayList<Comment>();
```

Comment:
nit: Let's use the more generic List for the definition, but keep ArrayList for the initialization. If we type this as a generic List, it can be used and applied more broadly. So something like: 

private List<Comment> comments = new ArrayList<Comment>();